### PR TITLE
Added conditions to editing address and added some styles

### PR DESCRIPTION
**Pull request by andrepestana-aot**
*Issue #:* /bcgov/entity5027

*Description of changes:*
-  Added conditions to addresses section to allow editing only when it is a correction
- Added some styling

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/YourCompany/OfficeAddresses.vue [hunk 1]
```diff
@@ -11,21 +11,21 @@
         <v-flex xs4>
           <label class="d-flex flex-wrap">
             <span class="mr-2">Mailing Address</span>
-            <v-chip v-if="mailingChanged"
+            <v-chip v-if="this.isCorrectionView() && mailingChanged"
```
Comment (cameron-freshworks) [inline src/components/YourCompany/OfficeAddresses.vue:14]: Within the template, it isn't required that you refer to this method with `this`.

File: src/components/YourCompany/OfficeAddresses.vue [hunk 2]
```diff
@@ -11,21 +11,21 @@
...
           />
-          <div v-else>(Not entered)</div>
+          <div v-else class="info-text">(Not entered)</div>
         </v-flex>
 
         <v-flex xs4>
           <label class="d-flex flex-wrap">
             <span class="mr-2">Delivery Address</span>
-            <v-chip v-if="deliveryChanged"
+            <v-chip v-if="this.isCorrectionView() && deliveryChanged"
```
Comment (cameron-freshworks) [inline src/components/YourCompany/OfficeAddresses.vue:28]: same as above.

**Review by cameron-freshworks — APPROVED** [review]
Looks good!
Just all the instances of `this` within the template could be removed but otherwise g2g.

File: src/components/YourCompany/OfficeAddresses.vue (hunk 1 above)
Comment (andrepestana-aot) [reply src/components/YourCompany/OfficeAddresses.vue]: Yes, you're right. Thanks!

**Comment by andrepestana-aot** [comment]
> Looks good!
> Just all the instances of `this` within the template could be removed but otherwise g2g.

Thanks! You can merge if it's ok!

**Review by cameron-freshworks — APPROVED** [review]

**Pull request merged**